client/my-sites/stats/activity-log/index.jsx

```diff
@@ -552,7 +552,7 @@ class ActivityLog extends Component {
 												applySiteOffset={ this.applySiteOffset }
 												disableRestore={ disableRestore }
 												disableBackup={ disableBackup }
-												isRewindActive={ 'active' === rewindState.state }
+												rewindState={ rewindState }
```

Comment:
instead of passing this object down through props can we instead connect `getRewindState()` into the `ActivityLogDay?

Reply:
Perfect, thanks for the thorough response. Will update this.